README.md

```diff
@@ -53,6 +53,212 @@ func main() {
 }
 ```
 
+## Running the Demo
+
+To run the demo, pull down the source code, install Go, and run `make test-verify`.
+This will use openssl to generate a certificate chain.
+
+During the in-toto verification process, `certificate constraints` are checked to ensure the build step link meta-data was signed with the correct SVID.
+
+## Building
+
+Download the source, run `make build`.
+
+## CLI
+
+```text
+Usage:
+  in-toto [command]
+
+Available Commands:
+  help        Help about any command
+  key         Key management commands
+  record      Creates a signed link metadata file in two steps, in order to provide evidence for supply chain steps that cannot be carried out by a single command
+  run         Executes the passed command and records paths and hashes of 'materials'
+  sign        Provides command line interface to sign in-toto link or layout metadata
+  verify      Verify that the software supply chain of the delivered product
+
+Flags:
+  -h, --help                              help for in-toto
+
+Use "in-toto [command] --help" for more information about a command.
+```
+
+### key
+
+```text
+Key management commands
+
+Usage:
+  in-toto key [command]
+
+Available Commands:
+  id          Output the key id for a given key
+  layout      Output the key layout for a given key
+
+Flags:
+  -h, --help   help for key
+
+Use "in-toto key [command] --help" for more information about a command.
+```
+
+### run
```

Comment:
Exclude should be added. Also, needs to reflect `--output-directory` -> `--metadata-directory` change. Applies to record as well.

Reply:
Changed to `--metadata-directory` for both run and record. `--exclude` is already added for both run and record.